src/Client/components/ques/WrittenQues.tsx

```
@@ -239,63 +256,73 @@
+                                                    question.id,
+                                                    e.target.checked
+                                                        ? false
+                                                        : question.isShortAnswer,
+                                                    e.target.checked
+                                                )
+                                            }
+                                            isDisabled={question.isShortAnswer}
+                                        >
+                                            Long Answer
+                                        </Checkbox>
+                                    </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)